src/driver.rs

```diff
@@ -366,3 +352,140 @@ impl MajorObject for Driver {
 }
 
 impl Metadata for Driver {}
+
+/// A wrapper around `GDALDriverManager`.
+/// This struct helps listing and registering [`Driver`]s.
+pub struct DriverManager;
+
+impl DriverManager {
+    /// Returns the number of registered drivers.
+    ///
+    /// # Example
+    ///
+    /// ```rust, no_run
+    /// use gdal::Driver;
+    /// println!("{} drivers are registered", Driver::count());
```

Comment:
If `Driver` is to be deprecated, should these examples be updated?

Reply:
[b16c050](https://github.com/georust/gdal/pull/324/commits/b16c0501aae2d5e9fb4c357a4ccf567441ad1fcd)